src/script/components/MessagesList/Message/ContentMessage/asset/ImageAsset/ImageAsset.tsx

```diff
@@ -105,8 +105,15 @@ export const ImageAsset = ({
 
   const imageAsset: CSSObject = {
     aspectRatio: isFileSharingReceivingEnabled ? `${asset.ratio}` : undefined,
-    maxWidth: asset.width,
-    maxHeight: imageUrl ? '80vh' : asset.height,
```

Comment:
replacing this by `maxHeight: !imageUrl ? 'asset.height' : '80vh',` does not fix the issue